src/main/java/seedu/address/logic/commands/DeleteLinkCommand.java

```diff
@@ -51,8 +51,9 @@ public CommandResult execute(Model model) throws CommandException {
 
         Person patient = model.getPerson(patientNric);
         Person caregiver = model.getPerson(caregiverNric);
-        requireNonNull(patient, PERSON_NOT_FOUND);
-        requireNonNull(caregiver, PERSON_NOT_FOUND);
+        if (patient == null || caregiver == null) {
+            throw new CommandException(PERSON_NOT_FOUND);
```

Comment:
Maybe you would want to have a separate name for this sort of exception? Because it is good to maintain a separate error message to the caller than if they enter an invalid person. 

You also may want to specifically mention if the caregiver is null or the patient is null in order to give a more fine grained error message

Reply:
i think i would prefer keeping it as a command exception to maintain consistency.

Regarding the fine grain, thanks for the suggestion.